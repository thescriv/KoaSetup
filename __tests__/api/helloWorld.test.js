const { startApi, stopApi } = require("../../src/api")
const {apiClient} = require("../apiClient")

let client

describe("HelloWorld API", () => {
  beforeAll(async () => {
    await startApi(3000)

    client = new apiClient(3000)
  })

  afterAll(async () => {
    await stopApi()
  })

  describe("HelloWorldApi", () => {
    test("do fetch helloWorldRouter", async () => {
      const { body, status } = await client.helloWorldApi()

      expect({ body, status }).toMatchSnapshot()
    })
  })
})
