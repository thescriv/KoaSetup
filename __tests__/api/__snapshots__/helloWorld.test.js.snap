// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloWorld API HelloWorldApi do fetch helloWorldRouter 1`] = `
Object {
  "body": Object {
    "message": "Hello World",
  },
  "status": 200,
}
`;
